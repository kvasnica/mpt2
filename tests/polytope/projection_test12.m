function projection_test12

% tests the improved heuristics:
%
% also we have had an error with verbose=2, becuase the mplp-based method was
% not described.

load projection_test11

% note that there is something wrong with CDD in this example (projection=3). it
% very often crashes matlab.
Q = projection(P, dim, struct('verbose', 2, 'projection', [4 7 6]));
