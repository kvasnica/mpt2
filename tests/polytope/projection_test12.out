PROJECTION: ESP failed, trying mplp-based method...
PROJECTION: mplp-based method failed, trying Fourier-Motzkin (mex version, fast)...
